hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/ddl/HMSDDLExecutor.java

```diff
@@ -247,6 +248,26 @@ public void dropPartitionsToTable(String tableName, List<String> partitionsToDro
     }
   }
 
+  @Override
+  public void updateTableComments(String tableName, Map<String, ImmutablePair<String,String>> alterSchema) {
+    try {
+      Table table = client.getTable(syncConfig.databaseName, tableName);
+      StorageDescriptor sd = table.getSd();
+      for (FieldSchema fieldSchema : sd.getCols()) {
```

Comment:
it will be better to a new sd

Reply:
`StorageDescriptor sd = new StorageDescriptor (table.getSd());`
like this?

Reply:
yes